contributing_to_docs/term_glossary.adoc

```diff
@@ -42,6 +42,16 @@ An action consists of _project_, _verb_, and _resource_:
 referenced resource itself, which can be a pod, deployment, build, etc.
 
 ''''
+=== Assisted Installer
```

Comment:
This is not in alphabetical order, can you move it down?

Reply:
Done, thanks!